scripts/mypy.bash

```diff
@@ -1,41 +1,50 @@
 #!/bin/bash
 # http://redsymbol.net/articles/unofficial-bash-strict-mode/
+
 set -o errexit
 set -o nounset
 set -o pipefail
 IFS=$'\n\t'
 
-image_name="$(basename "$0"):latest"
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+IMAGE_NAME="local/scripts-$(basename "$0"):latest"
+WORKDIR="$(pwd)"
+
+DEFAULT_MYPY_CONFIG="$(git rev-parse --show-toplevel)/mypy.ini"
+MYPY_CONFIG=$(realpath "${2:-${DEFAULT_MYPY_CONFIG}}")
 
-# NOTE: latest pydantic version (1.10.0) creates a invalid issue in mypy.
-# using mypy 0.971 (compiled: yes)
-# simcore_service_dynamic_sidecar/models/shared_store.py:12:47: error: Incompatible types in assignment (expression has type "List[_T]", variable has type "List[str]")
-docker buildx build --tag "$image_name" --load - &>/dev/null <<EOF
-FROM python:3.9.12-slim-buster
-RUN pip install --upgrade pip \
-    && pip install mypy==0.971 \
-                  pydantic[email]==1.9.2 \
-                  types-aiofiles==0.8.10 \
-                  types-PyYAML==6.0.11 \
-                  types-ujson \
-                  types-setuptools
-ENTRYPOINT ["mypy"]
-EOF
+Build() {
```

Comment:
out of curiosity why are the functions capital letters?

Reply:
I just followed how is done in the other bash scripts. I am happy to adopt another code-styling for bash

I found https://google.github.io/styleguide/shellguide.html. perhaps we should adopt it?